astropy/visualization/wcsaxes/coordinate_range.py

```diff
@@ -57,6 +58,15 @@ def find_coordinate_range(transform, extent, coord_types, coord_units, coord_wra
 
     ranges = []
 
+    new_coord_wrap = []
+    for coord_wrap in coord_wraps:
+        if isinstance(coord_wrap, Quantity):
+            new_coord_wrap.append(coord_wrap.to_value(u.deg))
+        else:
+            new_coord_wrap.append(coord_wrap)
```

Comment:
To avoid ``append`` and intermediate variables, can be simplified to a ternary operator in a list comprehension.

```
coord_wraps = [(c.to_value(u.deg) if isinstance(c, Quantity) else c) for c in coord_wraps]
```